### DISCUSSION TITLE
How to set 2 actor dispatchers

### DISCUSSION THREAD

**Discussion by mstghafouri** [fedify-dev/fedify#138] [Q&A]
I am trying to set up 2 actor dispatchers with different paths, one for users and one for podcasts

`federation.setActorDispatcher("/users/{handle}", async (ctx, handle) => {...})`

`federation.setActorDispatcher("/podcasts/{handle}", async (ctx, handle) => {...})`

What are best practices ?
Any help would be appreciated

**Comment by dahlia** [comment]
We are sorry.  Unfortunately, Fedify currently does not support multiple actor dispatchers.  😢
